### PR TITLE
Fix no chunked attr

### PR DESCRIPTION
**Pull request by elnuno**
I added a simple fix for #137 and a test that will only run on old versions of requests. Sorry about the revert commits, I can squash locally and re-submit if it helps.

### CODE REVIEW

File: tests/test_regressions.py [hunk 1]
```diff
@@ -29,3 +29,18 @@ def test_getattr_during_gc():
...
+
+
+def test_handle_no_chunked_attr():
+
+    class NoChunked(CacheControlAdapter):
+        def build_response(self, request, response, from_cache=False,
+                           cacheable_methods=None):
+            if hasattr(response, 'chunked'):
+                pytest.skip('Requests is new enough, test makes no sense.')
+                # delattr(response, 'chunked')
```
Comment (ionrock) [inline tests/test_regressions.py]: Can you remove this comment?

File: tests/test_regressions.py [hunk 2]
```diff
@@ -29,3 +29,18 @@ def test_getattr_during_gc():
...
         s.x
+
+
+def test_handle_no_chunked_attr():
+
+    class NoChunked(CacheControlAdapter):
+        def build_response(self, request, response, from_cache=False,
+                           cacheable_methods=None):
+            if hasattr(response, 'chunked'):
+                pytest.skip('Requests is new enough, test makes no sense.')
```
Comment (ionrock) [inline tests/test_regressions.py]: I might reword this as `Requests version supports chucked. Skipping this test.`

File: tests/test_regressions.py [hunk 3]
```diff
@@ -29,3 +29,18 @@ def test_getattr_during_gc():
...
+                # delattr(response, 'chunked')
+            return super().build_response(request, response, from_cache,
+                                          cacheable_methods)
+    sess = Session()
+    sess.mount('http://', NoChunked())
+    sess.get('http://httpbin.org/cache/60')
```
Comment (ionrock) [inline tests/test_regressions.py]: You need to assert something here! :)

**Comment by erayerdin** [comment]
I will refer to #137 so that everyone can find this PR easily.
